README.md

```diff
@@ -1,2 +1,68 @@
-# famly
-Gesellenstück Neue Fische 
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
This Readme and the available scripts noticed belongs to the client package.json. 
Maybe move this file and create a new README.md for the top-level scripts.

Reply:
Thank you for your review. I moved this file and created a new one for the top-level scripts.